src/components/Modal/EditSubkeywordModal.jsx

```diff
@@ -0,0 +1,191 @@
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+
+import asyncEditSubkeyword from "../../api/keyword/asyncEditSubkeyword";
+import { ERROR_MESSAGE, MODAL_TYPE } from "../../config/constants";
+import useBoundStore from "../../store/client/useBoundStore";
+import CreateKeywordButton from "../Button/CreateKeywordButton";
+import KeywordChip from "../Chip/KeywordChip";
+import Portal from "../Common/Portal";
+import PlusIcon from "../Icon/PlusIcon";
+import Label from "../UI/Label";
+import ModalBackground from "./ModalBackground";
+import ModalFrame from "./ModalFrame";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+
+const EditSubkeywordModal = () => {
+  const modalNode = document.getElementById("modal");
+  const userInfo = useBoundStore((state) => state.userInfo);
+  const { keywordId } = useParams();
+  const keywordName = "임시 대표 키워드 이름";
+  const [inputValue, setInputValue] = useState({
+    includedKeyword: "",
+    excludedKeyword: "",
+  });
+  const [errorMessage, setErrorMessage] = useState({
+    includedKeyword: "",
+    excludedKeyword: "",
+  });
+  const [subkeywordList, setSubkeywordList] = useState({
+    includedKeyword: ["추가 키워드1", "추가 키워드2"],
+    excludedKeyword: ["제외 키워드1", "제외 키워드2"],
+  });
+  const addModal = useBoundStore((state) => state.addModal);
+  const closeModal = useBoundStore((state) => state.closeModal);
+  const queryClient = useQueryClient();
+
+  const editSubkeywordMutation = useMutation({
+    mutationFn: (subkeywordInfo, keywordId) => asyncEditSubkeyword(subkeywordInfo, keywordId),
```

Comment:
아래 77번째 줄에서 아래와 같이 객체로 mutationFn에 인수로 넣어줬기 때문에, 아래와 같이 객체로 받아야 합니다 

```suggestion
    mutationFn: ({ subkeywordInfo, keywordId }) => asyncEditSubkeyword(subkeywordInfo, keywordId),
```

```js
editSubkeywordMutation.mutate(
      { subkeywordInfo, keywordId },
```
![image](https://github.com/user-attachments/assets/6d42b603-e625-4417-9bab-5b8b5e17cc5f)

Reply:
하나의 객체로 매개변수를 전달해야 하는데 누락이 있었어요. 아직 연결하지 못해 테스트 못한 부분인데, 먼저 발견해주셔서 감사합니다. 다음 부터는 놓치지 않을게요! 백엔드 API 작업 먼저 해서 테스트하거나 안전장치를 걸어두는게 좋겠어요 🤔